about/policy/data-sharing-agreement/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: State Geographic Information Database (SGID) Data Sharing Agreement
+layout: page_nocrumb
+author:
+  display_name: UGRC Office
+  email: ugrc@utah.gov
+tags:
+  - license
+  - policy
+categories: []
+date: 2020-01-25 17:00:00 -0700
+---
+
+The following Data Sharing Agreement applies to all individuals and organizations sharing data with the SGID in any way.
+
+`<Four different levels of getting agreement/acknowledgement, from easiest to most onerous>`
```

Comment:
Does this imply we will only accept new SGID contributions via Porter issues completed by the steward?

Reply:
I think so. If Porter's purpose is to ensure that additions/removals are tracked and that all the necessary steps are taken/wired up, accepting data outside of Porter kind of defeats the whole purpose.

Reply:
Or were you focusing on the "completed by the steward" part?

Reply:
this
> Or were you focusing on the "completed by the steward" part?

Reply:
I'm thinking even if we have the statute authority to add any state-created data, it's good neighborliness to work with the steward. If I'm tracking what you're implying, this means we can't open/complete an issue on their behalf, which introduces the hurdle of them knowing github to the process.

If that's the problem, maybe we can add a provision for an emailed acknowledgement of the agreement.

Reply:
Yes, we certainly want to and will work with them. My main concern is making a steward complete a Porter issue if they're uncomfortable with it so, yes, some provision for accepting the agreement in an alternative manner.

Reply:
I've updated the text; let me know your thoughts.

Reply:
I like it, thank you. Made another small suggestion to eliminate slight redundancy.